apps/ledger-live-mobile/src/screens/Portfolio/index.tsx

```diff
@@ -44,6 +44,8 @@ import CheckTermOfUseUpdate from "../../components/CheckTermOfUseUpdate";
 import TabBarSafeAreaView, {
   TAB_BAR_SAFE_HEIGHT,
 } from "../../components/TabBar/TabBarSafeAreaView";
+import CustomImageBottomModal from "../../components/CustomImage/CustomImageBottomModal";
+import Button from "../../components/Button";
```

Comment:
Both import are unused ? Is it for future usage ?

Reply:
ah nope I forgot to remove that